src/entities/FactorOperationalMemory.jl

```diff
@@ -42,6 +39,12 @@ struct CalcFactor{T <: AbstractFactor, M, P <: Union{<:Tuple,Nothing,AbstractVec
   _allowThreads::Bool
   """ user cache of arbitrary type, overload the [`preambleCache`](@ref) function. NOT YET THREADSAFE """
   cache::C
+
+  ## TODO Consolidation WIP with FactorMetadata
+  # full list of variables connected to the factor
+  fullvariables::Vector{DFGVariable}
```

Comment:
ah dynamic types.  Let's do Vector just to get rid of FMd and the come back for `{T<:Tuple}`?

Reply:
Yes, currently it’s dispatched dynamically in any case and this way it at least saves on compile time.
But the plan is to figure out the best way after the cleanup and not waste time now.